pkg/package-server/provider/inmem.go

```diff
@@ -222,11 +225,26 @@ func (m *InMemoryProvider) syncCatalogSource(obj interface{}) error {
 
 		log.Debugf("storing packagemanifest at %+v", key)
 		m.manifests[key] = manifest
+		m.out <- manifest
```

Comment:
Won't this cause every single package to stream to the client during every resync of CatalogSources?